orc8r/cloud/deploy/bare-metal-ansible/roles/magma/fluentd/templates/fluentd-values.yaml.j2

```diff
@@ -0,0 +1,67 @@
+configMaps:
+  forward-input.conf: |-
+    <source>
+      @type forward
+      port 24224
+      bind 0.0.0.0
+      <transport tls>
+        ca_path /certs/certifier.pem
+        cert_path /certs/fluentd.pem
+        private_key_path /certs/fluentd.key
+        client_cert_auth true
+      </transport>
+    </source>
+  output.conf: |-
+    <match **>
```

Comment:
this one is missing EventD match introduced resently
https://github.com/magma/magma/blob/86e1603f898188f3315d40c17da1c57eed19b768/orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf#L58